docker/archive/dest.go

```diff
@@ -19,17 +19,32 @@ func newImageDestination(ctx *types.SystemContext, ref archiveReference) (types.
 	if ref.destinationRef == nil {
 		return nil, errors.Errorf("docker-archive: destination reference not supplied (must be of form <path>:<reference:tag>)")
 	}
-	fh, err := os.OpenFile(ref.path, os.O_WRONLY|os.O_EXCL|os.O_CREATE, 0644)
+	// tries to open a file first to accomodate for load when using stdout
+	// as the file is created first then written to
+	// if the file doesn't exist then it creates a new file for exclusive use
+	// if the file exists, it checks to ensure the size is 0
```

Comment:
Right; containers/image only works one image at a time. There is no “list tags” (other than a single very ugly hack via `docker/Image.GetRepositoryTags`), there is no “copy several images”.

(IMHO the “copy all tags of an image ~ all versions of the application” functionality is not all that useful… but I have no idea what real-world customers do. The ability to copy several differently-named images into a single archive, sharing a single copy of the base layers, is more useful OTOH. Still, because the docker archives discard manifest digests, using a centralized registry server seems more generally useful than copying tar files around using… ssh?)